### PR TITLE
fix(sentry): Requirement name

### PR DESCRIPTION
**Pull request by tigattack**
# Initial Checklist
- [x] Has @tigattack / @issy been added as a reviewer?
- [x] If applicable, have the relevant project(s), milestone(s), and label(s) been applied?
- [ ] If applicable, have you added details of the cog to the readme as per [README.md](https://github.com/rHomelab/LabBot-Cogs/blob/develop/README.md#cog-summaries)?

# Details
**Does this resolve an issue?**
No

### Features / Fixes
* Fix typo in Sentry requirements.

### Breaking Changes
N/A.

### CODE REVIEW

File: sentry/info.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "sentry"
     ],
     "requirements": [
-        "sentry-sdk"
+        "sentry_sdk"
```
Comment (issy) [inline sentry/info.json:13]: [`sentry-sdk`](https://pypi.org/project/sentry-sdk/) is the correct name for the requirement. `sentry_sdk` just converts itself to `sentry-sdk` when you install with pip

I've tested the cog installation and there doesn't seem to be any issue with it installing the correct package and loading the cog

Comment (tigattack) [reply sentry/info.json:13]: Hmm, it wasn't installing correctly when I tested and this seemed to solve it 🤔 I'll do some more testing.

Comment (issy) [reply sentry/info.json:13]: Interesting. I'll try to replicate, but there *shouldn't* be an issue.
Have you tried adding the repo and installing the cog from scratch?

Comment (BeryJu) [reply sentry/info.json:13]: So the main reason this started is me testing it on a redbot instance, and Red Not installing the package as described in the info json.

**Comment by BeryJu** [comment]
Closed because issy is correct and sentry-sdk is the correct package name

**Pull request closed without merging**